app/views/apps/index.html.haml

```diff
@@ -1,11 +1,14 @@
 - content_for :title, t('.title')
 - content_for :action_bar do
-  %span= link_to(t('.new_app'), new_app_path, :class => 'add') if current_user.admin?
+  - if current_user.admin?
+    = link_to new_app_path, :class => 'btn btn-success' do
+      %i.glyphicon.glyphicon-plus-sign
+      %span= t('.new_app')
 
-%table.apps
+%table.table-striped.table-hover.apps
   %thead
     %tr
-      %th= t('.name')
+      %th.main-column= t('.name')
```

Comment:
Please check your text-alignments, because *errors* and *last_deploys* cell content are in the center, but headers are not...